.circleci/config.yml

```diff
@@ -234,6 +234,8 @@ jobs:
                   key: coverage-python-sra-client-{{ .Environment.CIRCLE_SHA1 }}
                   paths:
                       - ~/repo/python-packages/sra_client/.coverage
+            - store_artifacts:
+                  path: ~/repo/packages/abi-gen-wrappers/generated_docs
```

Comment:
This is in the wrong spot.  You've got it persisting artifacts generated as part of `test-python`, but the docs are generated over in `build`, not `test-python`.  This addition needs to move over to be part of `build`.  (Concretely, I don't see any `abi-gen-wrappers` artifacts available in the `test-python` build results.)